calculator/README.md

```diff
@@ -1,4 +1,4 @@
-# AWESOME-BOOKS | MODULE-II
+# MATH MAGICIANS | MODULE-III
 
```

Comment:
There is a typo on your instructions to run the project 🧐 In lines 50-51, the correct command is `npm start` rather than `npm started`. Kindly make this little change so that your instructions are on point 🔥